pkg/audit/manager.go

```diff
@@ -758,11 +764,18 @@ func (am *Manager) addAuditResponsesToUpdateLists(
 		}
 
 		totalViolationsPerEnforcementAction[ea]++
+		err := am.pubsubSystem.Publish(context.Background(), "audit-connection", "audit-topic", StatusViolation{Group: gvk.Group, Version: gvk.Version, Kind: gvk.Kind, Name: r.Constraint.GetName(), Namespace: r.Constraint.GetNamespace(), Message: r.Msg, EnforcementAction: string(ea)})
```

Comment:
Still working on making `audit-connection` and `audit-topic` configurable for user